OracleJava/8/serverjre/Dockerfile

```diff
@@ -31,23 +31,21 @@ LABEL maintainer="Aurelio Garcia-Ribeyro <aurelio.garciaribeyro@oracle.com>"
 RUN set -eux; \
 	yum install -y \
 		gzip \
-		tar \
-	; \
-	rm -rf /var/cache/yum
+		tar;
```

Comment:
This could be simplified:

```Dockerfile
RUN yum install -y gzip tar
```

Reply:
Updated the requested dockerfiles and the other similar ones.